RPIPs/RPIP-53.md

```diff
@@ -0,0 +1,192 @@
+author: Joe Clapis (@jcrtp), Jacob Shufro (@jshufro)
+discussions-to: https://dao.rocketpool.net/t/rewards-tree-spec-v10/2937
+status: Review
+type: Protocol
+created: 2024-04-19
+---
+
+
+## Abstract
+
+The incoming Houston upgrade introduces [RPIP-31](./RPIP-31.md) - the **RPL withdrawal address**. This is a supplementary setting, different from the "regular" withdrawal address used in Atlas and below, that node operators can opt into. It's intended to specify a "secondary" withdrawal address that the node's RPL rewards will be sent to, whereas the node's ETH rewards will be sent to its "regular" withdrawal address (known in Houston as a **primary** withdrawal address).
+
+Realistically, most node operators won't end up using this feature. It's only really helpful for node operators that don't supply their own RPL when creating a minipool. If you don't fall into this camp, rest assured that *this proposal will have no impact on you*.
+
+Unfortunately, node operators that *do* use this new feature can **no longer trigger rewards claims** from the Merkle rewards system. More specifically they can distribute minipool rewards from the Beacon Chain, but can't claim the rewards produced every 28 days via the Merkle rewards system. That's not a problem for RPL rewards, since the RPL withdrawal address gets those anyway, but the issue is this includes Smoothing Pool ETH rewards as well. Neither the node nor the address that would get the Smoothing Pool ETH (the primary withdrawal address) can actually trigger a claim; *only* the RPL withdrawal address will be able to do it.
+
+This can't be fixed for nodes that opt into setting an RPL withdrawal address if they also use their node as their primary withdrawal address because of technical reasons described in the [Rationale](#rationale) section of the RPIP. However, it *can* be fixed for nodes that opt into it and use an address *other* than their node as the primary withdrawal address. I anticipate that most nodes opting into the RPL withdrawal address feature will fall into this camp.
+
+This RPIP proposes to achieve this by generalizing the rewards tree, changing *nodes* to *claimers*. The tree will now specify the amount of RPL and Smoothing Pool ETH available to individual claimers, where:
+- If a node does not have an RPL withdrawal address set, **or** if it has one but its primary withdrawal address is the same as the node address, their node address is the claimer address (same as today).
+- If a node has an RPL withdrawal address set and its primary withdrawal address is different from the node address, the node's rewards will be split in two:
+  - One claimer will be the node's primary withdrawal address, which will have the node's ETH portion of the rewards.
+  - The other will be the node's RPL withdrawal address, which will have the node's RPL portion of the rewards.
```

Comment:
I'd probably keep this much simpler and point to Motivation for details.

> In the initial implementation there was an unforeseen issue where sometimes the parties receiving rewards would not be able to execute the claims to get them and would rely on another party (see (Motivation)[#motivation) for details). This RPIP proposes to mitigate the problem as much as possible by generalizing the rewards tree...

Reply:
Ok, shortened the abstract - hopefully it reads better now.